libvast/fbs/partition_transform.fbs

```diff
@@ -0,0 +1,23 @@
+include "uuid.fbs";
+
+namespace vast.fbs.partition_transform;
+
+// TODO: Does it make sense to also store the id of
+// the transform itself and the `keep_input` flag?
+table v0 {
+	input_partitions: [UUID];
```

Comment:
Can you add a comment that this is empty if preserve-input is enabled for compaction?